src/ffi.rs

```diff
@@ -0,0 +1,91 @@
+extern crate libc;
+
+#[allow(non_camel_case_types)]
```

Comment:
Not sure if you care, but (I THINK) you can probably shove this in lib to prevent explicitly stating this for all your types.

Reply:
You might prefer to write `#![allow(non_camel_case_types)]` at the top of the `ffi` module (in `rill-rs`) or at the crate's root instead of `#[allow(non_camel_case_types)]` on every type that needs it.